### PR TITLE
Clean up docs and shorten README

### PR DESCRIPTION
**Pull request by aelligp**
This PR addresses #204 and cleans some rendering errors in the docs & links the pkg.

### CODE REVIEW

**Comment by aelligp** [comment]
@luraess

File: README.md [hunk 1]
```diff
@@ -59,268 +60,10 @@ julia> ]
...
-
 ## Miniapps
 
-Currently there are 4 convection miniapps with particles and 4 corresponding miniapps without. The miniapps with particles are:
-
-  * [Layered_convection2D.jl](miniapps/convection/Particles2D/Layered_convection2D.jl)
-  * [Layered_convection2D_nonDim.jl](miniapps/convection/Particles2D_nonDim/Layered_convection2D.jl)
-  * [Layered_convection3D.jl](miniapps/convection/Particles3D/Layered_convection3D.jl)
-  * [WENO_convection2D.jl](miniapps/convection/WENO5/WENO_convection2D.jl)
+Available miniapps can be found [here](miniapps) and will be updated regularly. The miniapps are designed to be simple and easy to understand, while still providing a good basis for more complex applications. The miniapps are designed to be run on a single node, but can be easily extended to run on multiple nodes using [ImplicitGlobalGrid.jl](https://github.com/omlins/ImplicitGlobalGrid.jl) and [MPI.jl](https://github.com/JuliaParallel/MPI.jl).
```
Comment (luraess) [inline README.md:65]:
```suggestion
Available miniapps can be found in the [miniapps folder](miniapps) and will be updated regularly. The miniapps are designed to be simple and easy to understand, while still providing a good basis for more complex applications. The miniapps are designed to be run on a single node, but can be easily extended to run on multiple nodes using [ImplicitGlobalGrid.jl](https://github.com/omlins/ImplicitGlobalGrid.jl) and [MPI.jl](https://github.com/JuliaParallel/MPI.jl).
```

**Comment by luraess** [comment]
@aelligp seems that some files like code of conduct, contributing, authors are duplicated and not properly rendered - needs a fix.

**Comment by aelligp** [comment]
>  code of conduct, contributing, authors are duplicated and not properly rendered

Code of conduct, authors, etc are generated from files in the root of the repo. 
The rendering issues for contributions.md are fixed. License and code of conduct remain indented as it is a copy of official documents (see [Trixi.jl](https://trixi-framework.github.io/Trixi.jl/stable/code_of_conduct/))

**Pull request merged**